ciscript

```diff
@@ -37,18 +37,14 @@ cd ../build
 export PATH="`pwd`/install/bin:`pwd`/bin:$PATH"
 make install
 
-cp ../test/Makefile ./TestMakefile
+TEST_DIR=../test
 
-make -f TestMakefile -j`nproc`
+lit $TEST_DIR
 
-diff io_output1.txt ../test/ioTest1.output
-diff `ls | grep io_log` ../test/ioTest3.output
+cd $TEST_DIR/defn
+diff io_output1.txt ../ioTest1.output
```

Comment:
These checks can probably be inlined into the respective test now.

Reply:
Will do.